starter/script.js

```diff
@@ -1 +1,176 @@
-// put code here!
+// one hipster's tale
+
+// functions:
+function getAdventure(adventureState) {
+//   input: adventureState, player info
+//   effect: display current adventure
+  console.log(adventureText[adventureState]);
+};
+
+function getResponse(adventureState) {
+//  input: question/response key, playerinfo
+//  returns: option letter
+  getAdventure(adventureState);
+  var a = '';
+  console.log(adventureQ[adventureState]);
+  while (a != 'a' && a != 'b' && a != 'c') {
+    a = prompt(adventureChoices[adventureState]);
+  };
+  if (a === 'a') {
+    player['hipsterIndex'] += 3;
+  }
+  return a;
+};
+
+// game text/info:
+var playerTypes = {
+  'a': 'Foodie',
+  'b': 'Hipster',
+  'c': 'Crafter',
+  'd': 'Normcore'
+};
+
+var adventureText = {
+  'greeting': '"Hi there! I\'m Jans. Welcome to square."\nA small man with elaborate facial hair and a plaid shirt stands before you.\nLooks like you\'re new around here. This is a pop-up art space incubator slash maker innovation lab slash artisan food market."\nPeople mill about around you, sipping beers and looking at tasteful home goods.',
+  '1': 'It looks like there are lots of things to see around here. Some sort of bicycle race/Instagram contest is taking place in the outdoor plaza before you. Live music is drifting from a small stage. There are also some food vendors set up in tents.',
```

Comment:
I really like how you are able to access properties in these 'Text, Question, & Choices' objects based on the current `adventureState`. Another nit-picky refactoring opportunity is to combine them all into a single object. I found myself jumping back and forth a bunch between objects in reading this code. I think it might help the readability to have all the data for a specific state in the same location, doing something like this:
```js
var adventure = {
  1a:  {
    text: 'some text',
    q: 'some question',
    choice: 'some choice'
  }
}
```